lib/curl_gssapi.c

```diff
@@ -66,14 +66,12 @@ OM_uint32 Curl_gss_init_sec_context(
   if(data->set.gssapi_delegation & CURLGSSAPI_DELEGATION_FLAG)
     req_flags |= GSS_C_DELEG_FLAG;
 
-  /*retrieve imposonated credential on Linux/Unix*/
+  /*retrieve impersonated credential on Linux/Unix*/
   gss_cred_id_t cred;
   cred = GSS_C_NO_CREDENTIAL;
   if(data->set.delegcred != NULL)
     cred = (gss_cred_id_t)data->set.delegcred;
```

Comment:
If this is always a `gss_cred_id_t`, why isn't the `data->set.delegcred` struct field using that type?

Reply:
See my comment regarding SSPI. Make sense to have it as `void *`.